test/dynamo/test_cudagraphs_expandable_segments.py

```diff
@@ -16,9 +16,10 @@
 from dynamo.test_cudagraphs import TestAotCudagraphs  # noqa: F401
 
 
-REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent.parent
+REPO_ROOT = Path(__file__).absolute().parents[2]
 
 sys.path.insert(0, str(REPO_ROOT))
+
 from tools.stats.import_test_stats import get_disabled_tests
```

Comment:
Looks like internal failure on this line: 
```ModuleNotFoundError: No module named 'tools'```

Reply:
I guess the path insert above doesn't work anymore.
cc @eellison why is this needed? We really shouldn't play with python path to find tests?